pkg/crc/api/api_test.go

```diff
@@ -79,10 +79,10 @@ func TestApi(t *testing.T) {
 		{
 			command:       "start",
 			clientFailing: true,
-			args:          json.RawMessage(`{"pull-secret":"/Users/fake/pull-secret"}`),
+			args:          json.RawMessage(`{"pullSecretFile":"/Users/fake/pull-secret"}`),
 			expected: map[string]interface{}{
 				"Status":         "",
-				"Error":          "Incorrect arguments given: json: unknown field \"pull-secret\"",
+				"Error":          "Failed to start",
```

Comment:
This was testing the code path where a wrong argument is passed, the next test also tests at [L#100](https://github.com/code-ready/crc/pull/2342/commits/9e6e49b9f06c592d7716cde0e4a702a5248d2395#diff-ccfc08dc0022526bf42f302553c3525c97523de6be15f9c9ce4d895e4221f342L100) also tests the same thing